pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/PersistentTopicsTest.java

```diff
@@ -1148,4 +1149,21 @@ public void testAdminTerminatePartitionedTopic() throws Exception{
             Assert.assertEquals(e.getMessage(), "Termination of a non-partitioned topic is not allowed using partitioned-terminate, please use terminate commands.");
         }
     }
+
+    /**
+     * Fix issue 13573
+     */
+    @Test
+    public void testTopicReferenceNPEReturn404(){
+        String topicName = "notFoundTopic";
+        String exceptionDetail = "testTopicReferenceNPEReturn404";
+        doThrow(new RuntimeException(exceptionDetail)).when(pulsar).getBrokerService();
+        try{
+            persistentTopics.getTopicReference(TopicName.get(topicName));
+        }catch (Exception ex){
+            Assert.assertTrue(ex instanceof RestException);
+            Assert.assertTrue(((String)((RestException) ex).getResponse().getEntity()).contains(exceptionDetail));
+        }
```

Comment:
```suggestion
        try {
            persistentTopics.getTopicReference(TopicName.get(topicName));
        } catch (Exception ex) {
            Assert.assertTrue(ex instanceof RestException);
            Assert.assertTrue(((String)((RestException) ex).getResponse().getEntity()).contains(exceptionDetail));
        }
```

Fix the code style. BTW, I'm wondering why the checkstyle plugin doesn't check the blanks.